lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -32,7 +32,7 @@
  * @property {Run[]} runs
  */
 
-import {strict as assert} from 'assert';
+import assert from 'assert/strict';
```

Comment:
`'assert/strict'` also added in Node 15 :)

Reply:
Y'all could maybe beef up the PR descriptions because it took me a bit of debugging to figure out what was going wrong for me is apparently what is being fixed here :P

This is from `yarn build-smokehouse-bundle` failing on polyfilling `assert.strict`, but we only `yarn build-all` in `ci.yml`, which is Node 14 only, so the action wasn't failing?

Ideally we could get [their polyfill](https://github.com/FredKSchott/rollup-plugin-polyfill-node/blob/9097b194f078dd9c7441a0510dd4baea8dfcc283/polyfills/assert.js) updated if we're going to keep using it (even [browserify's polyfill](https://github.com/browserify/commonjs-assert/blob/bba838e9ba9e28edf3127ce6974624208502f6bc/assert.js#L600-L609) has `strict`), but for now we could switch to just `assert` but manually use `assert.strictEqual` and `assert.deepStrictEqual` instead of  `assert.equal` and `assert.deepEqual`. Not a huge deal.

Reply:
> This is from yarn build-smokehouse-bundle failing on polyfilling assert.strict

yup

> but we only yarn build-all in ci.yml, which is Node 14 only, so the action wasn't failing?

The GH action wasn't (because of wrong exit codes), correct. The import into google3 (see import_tool) was spitting out errors while bundling smokehouse but still chugging along.